cookbooks/aws-parallelcluster-environment/test/controls/cfnconfig_mixed_spec.rb

```diff
@@ -9,7 +9,7 @@
 # This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
 # See the License for the specific language governing permissions and limitations under the License.
 
-control 'cfnconfig_file_configuration' do
+control 'tag:install_cfnconfig_file_configuration' do
```

Comment:
should be tag:config not tag:install

Reply:
I thought its the install recipe which is why I made the mistake of moving it out of Config phase. Will make the relevant changes. Thanks.